dll/win32/browseui/explorerband.cpp

```diff
@@ -1329,8 +1329,8 @@ HRESULT STDMETHODCALLTYPE CExplorerBand::OnWinEvent(HWND hWnd, UINT uMsg, WPARAM
                     LPCITEMIDLIST pidlChild;
 
                     hr = SHBindToParent(info->absolutePidl, IID_PPV_ARG(IShellFolder, &pParent), &pidlChild);
-                if (!SUCCEEDED(hr) || !pParent.p)
-                    return E_FAIL;
+                    if (!SUCCEEDED(hr) || !pParent.p)
+                        return E_FAIL;
```

Comment:
Are there real changes here? Or is it just coding style changes?
This is hard to review.
If that's just coding style, I suggest to revert.
Furthermore, that's Wine code and will break further syncing